tests/testthat/test_Spectrum_Constructor.R

```diff
@@ -126,6 +128,30 @@ test_that("C-level multi-Spectrum1 constructor with M/Z ordering", {
         expect_identical(mz(spectL[[i]]), mzValsList[[i]][idxList[[i]]])
         expect_identical(intensity(spectL[[i]]), intValsList[[i]][idxList[[i]]])
     }
+
+    ## Check empty spectra.
+    ## Spectrum1
+    mzVals <- sort(abs(rnorm(200, mean = 100, sd = 10)))
```

Comment:
I don't believe that these tests will ever fail (fingers crossed), but I'll add the `set.seed` to be on the save side.